t/nutriscore.t

```diff
@@ -3,6 +3,8 @@
 use strict;
 use warnings;
 
+use utf8;
+
```

Comment:
why don't we have the use `Modern::Perl '2017';` 

shan't it be in every file ?

Reply:
I'll add it.